kube_log_watcher/main.py

```diff
@@ -165,7 +169,7 @@ def sync_containers_log_agents(
             logger.exception('Failed to sync log config with agent {}'.format(agent.name))
 
     # 4. return new containers, stale containers
-    return existing_container_ids, stale_container_ids
+    return new_container_ids, set(stale_container_ids)
```

Comment:
Isn't `stale_container_ids` a set?

Reply:
ähm yes. Forgot to remove it after debugging